shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -232,6 +237,58 @@ void FlutterWindowsEngine::SetPluginRegistrarDestructionCallback(
   plugin_registrar_destruction_callback_ = callback;
 }
 
+void FlutterWindowsEngine::SendWindowMetricsEvent(
+    const FlutterWindowMetricsEvent& event) {
+  if (engine_) {
+    embedder_api_.SendWindowMetricsEvent(engine_, &event);
+  }
+}
+
+void FlutterWindowsEngine::SendPointerEvent(const FlutterPointerEvent& event) {
+  embedder_api_.SendPointerEvent(engine_, &event, 1);
```

Comment:
do we need to null-check engine_ here? If not, is the one in SendWindowMetricsEvent necessary?

Reply:
Couldn't hurt; I couldn't easily convince myself that we couldn't have races around pointer event handling during teardown. Added.